src/session-feedback/dtos/session-feedback.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsNotEmpty, IsString } from 'class-validator';
+import { FEEDBACK_TAGS_ENUM } from 'src/utils/constants';
+
+export class SessionFeedbackDto {
+  @IsNotEmpty()
+  @IsString()
+  @ApiProperty({ type: String })
+  sessionFeedbackId: string;
+
+  @IsNotEmpty()
+  @IsString()
+  @ApiProperty({ type: String })
+  sessionId: string;
+
+  @IsNotEmpty()
+  @IsEnum(FEEDBACK_TAGS_ENUM)
+  @ApiProperty({
+    enum: FEEDBACK_TAGS_ENUM,
+    type: String,
+    example: Object.values(FEEDBACK_TAGS_ENUM),
```

Comment:
This will show all the enum values in the swagger "Example Value" section in a list - not sure if this is more or less convenient than it just saying "string"

Reply:
Nice!